api/handlers_test.go

```diff
@@ -19,17 +19,16 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
-var DiagnosticsBundleDir = "/tmp/snapshot-test"
-var testCfg *config.Config
 
-func init() {
-	if runtime.GOOS == "windows" {
```

Comment:
Just to be sure: using `ioutil.TempDir` you remove the need of OS dependent code, right?

Reply:
Correct :heavy_check_mark: